src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -65,129 +122,228 @@ export const WorkspaceList = ({ registeredUseCases$ }: WorkspaceListProps) => {
     [application, http]
   );
 
-  const searchResult = useMemo(() => {
-    if (queryInput) {
-      const normalizedQuery = queryInput.toLowerCase();
-      const result = workspaceList.filter((item) => {
-        return (
-          item.id.toLowerCase().indexOf(normalizedQuery) > -1 ||
-          item.name.toLowerCase().indexOf(normalizedQuery) > -1
-        );
-      });
-      return result;
-    }
-    return workspaceList;
-  }, [workspaceList, queryInput]);
+  const formatDate = function (dateString: string) {
```

Comment:
Better to move `formatDate`  function from this component and tests file to a utils file to maintain. Besides, OSD uses `moment` as dependency, could we use some format functions in moment directly?

Reply:
thanks for your advice